alliance_auth/urls.py

```diff
@@ -203,4 +204,18 @@
                        url(r'^notifications/$', 'notifications.views.notification_list', name='auth_notification_list'),
                        url(r'^notifications/(\w+)/$', 'notifications.views.notification_view', name='auth_notification_view'),
                        url(r'^remove_notifications/(\w+)/$', 'notifications.views.remove_notification', name='auth_remove_notification'),
-)
+
+                       # FleetActivityTracking (FAT)
+                       url(r'^fat/$', 'fleetactivitytracking.views.fatlink_view', name='auth_fatlink_view'),
```

Comment:
Haha I like SVEA.

Could be simply `pap`, as I know that's what gewnies call them. No idea what pap means or stands for.